Jenkinsfile

```diff
@@ -8,6 +8,7 @@ pipeline {
         }
         stage('Test') {
             steps {
+                sh 'docker pull linagora/tmail-backend:memory-branch-master'
```

Comment:
Do we need the same thing for gatling-imap?

Reply:
https://github.com/linagora/gatling-imap/pull/55

but as you approved I guess you saw it :)